fixmake.sh

```diff
@@ -0,0 +1,3 @@
+#grep "CC_FLAGS :=" pmufw/Makefile | grep -e "-Os" || sed -i '/-mxl-soft-mul/ s/$/ -Os -flto -ffat-lto-objects/' pmufw/Makefile
+cd pmufw
+make
```

Comment:
Can remove this file. Not necessary anymore

Reply:
Removed.